examples/integration-tutorials/sending-a-simple-message-to-a-service/sending-a-simple-message-to-a-service/README.md

```diff
@@ -0,0 +1,56 @@
+# sending-a-simple-message-to-a-service
```

Comment:
It would be better if we change the heading to something like "Sending a Simple Message to a Service"

Reply:
Updated.